### ISSUE TITLE
Godot crashes during import with 16bit images

### ISSUE THREAD

**Issue by nesja** [2shady4u/godot-kra-psd-importer#6]
Godot 3.2.1 crashes during import with 16bit images. 8bit works perfect with .kra but still crashes with .psd. I'll add a description if I can isolate the issue conditions more precisely. 

Thank you for the great plugin, it saves a lot of time.

**Comment by 2shady4u** [comment]
@nesja 

Indeed, the KRA part of this plugin can only handle 8-bit at the current time...

Regarding PSD... last I remember PSD should be supported for 8, 16 and 24 bit 🤔 
Does this crash happen when you attempt to process the example PSD in the demo project?

Cheers!

**DISCLAIMER**: This repo is very much an unfinished work-in-progress.. so while it does the job for simple examples, I do not recommend/endorse its use in production until it is in a later stage of development.

**Comment by nesja** [comment]
@2shady4u 
The example PSD works fine. Other PSDs are crashes regardless of depth, so probably cause of the issue is not associated with issue topic.
So I'll just use it carefully as it is. Thanks!
